src/CattyTests/Parser & Serialization/CatrobatLanguage0.991/XMLParserTests/Unit Tests/XMLParserTests0991.swift

```diff
@@ -558,4 +558,14 @@ class XMLParserTests0991: XMLAbstractTest {
         XCTAssertTrue(disabledBrick.isDisabled)
         XCTAssertFalse(enabledBrick.isDisabled)
     }
+
+    func testSpeakBrickWithoutCommentedOut() {
+        let project = self.getProjectForXML(xmlFile: "SpeakBrickWithoutCommentedOut_0991")
+        let object = project.objectList.object(at: 0) as! SpriteObject
+        let startScript = object.scriptList.object(at: 0) as! Script
+        let speakBrick = startScript.brickList.object(at: 0) as! SpeakBrick
+
+        XCTAssertEqual(speakBrick.formula.formulaTree.value!, "Panda", "Invaid brick")
```

Comment:
Thanks for the changes! You can leave this assert in here if you want, but please either remove the assert message (_"Invaid brick"_) or change it so that it reflects the current situation, i.e. "_Invalid formula value_" (I think this information is not necessary at all though).